components/org.wso2.carbon.siddhi.editor.core/src/main/java/org/wso2/carbon/siddhi/editor/core/internal/EditorMicroservice.java

```diff
@@ -931,4 +948,37 @@ protected void registerConfigProvider(ConfigProvider configProvider) {
     protected void unregisterConfigProvider(ConfigProvider configProvider) {
         this.configProvider = null;
     }
+
+    protected void getSampleFiles() {
```

Comment:
@kts-desilva can we change this name to "loadSampleFiles"

Reply:
Sure. Fixed in 43b4465c20aa7a2c50581e2401f9a2ac6fc7ebf0